dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First1.json

```diff
@@ -9,6 +9,10 @@
       "See? We told you!",
       "Venturing out of town is way too dangerous!\nDid you come back after realising that?"
     ],
+    "pl": [
+      "Widzisz? Mówiliśmy ci!",
+      "Chodzenia poza domem jest zbyt niebezpieczne!\nCzy wróciłaś, gdy zauważyłaś tego?"
```

Comment:
"Zapuszczanie się tak daleko od miasteczka jest zbyt niebezpieczne" I think it sounds a bit better